dlpy/applications/darknet.py

```diff
@@ -69,17 +70,27 @@ def Darknet_Reference(conn, model_table='Darknet_Reference', n_classes=1000, act
         that are larger than those sizes are cropped.
         Valid Values: 'none', 'unique', 'randomresized', 'resizethencrop'
         Default: 'unique'
+    random_mutation : string, optional
+        Specifies how to apply data augmentations/mutations to the data in the input layer.
+        Valid Values: 'none', 'random'
+        Default: 'NONE'
 
     Returns
     -------
     :class:`Sequential`
 
     '''
 
+    conn.retrieve('loadactionset', _messagelevel='error', actionset='deeplearn')
+
+    # get all the parms passed in
+    parameters = locals()
```

Comment:
I am not seeing any need for this. I hate using this call is it hides a lot of things. As this is an open source package I prefer to show everything as much as simple and open